hack/update-deps.sh

```diff
@@ -27,3 +27,15 @@ dep ensure
 
 rm -rf $(find vendor/ -name 'OWNERS')
 rm -rf $(find vendor/ -name '*_test.go')
+
```

Comment:
This should come before the `rm`s above.

Reply:
Done